source/unified-test-format/unified-test-format.rst

```diff
@@ -369,6 +369,13 @@ The structure of this object is as follows:
   "sharded" topology, test runners MUST accept any type of sharded cluster (i.e.
   "sharded" implies "sharded-replicaset", but not vice versa).
 
+- ``serverless``: Optional string. Whether or not the test should be run on
+  Serverless instances imitating sharded clusters. Valid values are "require",
```

Comment:
Unless "Serverless" is going to be a defined term or proper noun for branding, I think lowercase is fine. Note that it is lowercased in https://www.mongodb.com/serverless (unless that has yet to be updated for what this is referring to).

Reply:
Updated.